docs/blog/posts/2021-06-18-srun.md

```diff
@@ -1,11 +1,14 @@
 ---
-title: Using srun to Launch Applications Under Slurm
-postdate: June 18, 2021
-layout: post
-author: Tim Kaiser
-description: Describes options to get the desired layout of tasks to nodes/cores using srun.
+date: 2021-06-18
+authors:
+  - timkaiser
 ---
 
+# Using srun to Launch Applications Under Slurm
+
+This article primarily discusses options for the srun command to enable good parallel execution.
```

Comment:
I think this intro sentence should be the description that was deleted from the metadata. "This article describes options to get the desired layout of tasks to nodes/cores using srun."